.github/GOVERNANCE.md

```diff
@@ -0,0 +1,82 @@
+# Roles and responsibilities
+
+Kindly is maintained through volunteer work - all contributions are welcome and we look to invite new collaborators at all times.
+The roles described below are intended as guidelines for different ways to get involved and different levels of commitment you may choose to take on at any point.
+
+How much you contribute and how long for is entirely at your discretion and collaborators can change their roles in the community in any way at any time that is relevant to them.
+
+There are two simple steps to taking on any role:
+
+- 🔨 **Do** the work involved.  
+  This community is run on willful collaboration, meaning we invite and expect everyone to identify the areas of Kindly they would like to help with and take any reasonable action towards improving the platform.  
+  If you are interested in being a maintainer, find ways in which you can help maintainers with their tasks;  
```

Comment:
is the semi colon intentional here?

Reply:
I think it was but reading back this is weird so I've changed it a bit 😅